erpnext/stock/doctype/packing_slip_item/packing_slip_item.json

```diff
@@ -12,6 +12,10 @@
   "batch_no",
   "desc_section",
   "description",
+  "warehouse_details_section",
```

Comment:
Kindly refer the image.
![packing_slip_item_fields](https://user-images.githubusercontent.com/80836439/123172038-296d1980-d49a-11eb-8d9e-f93ba39824e8.png)

https://user-images.githubusercontent.com/80836439/123172346-a5676180-d49a-11eb-838f-2a3443285ed8.mp4